libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -94,6 +114,10 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
  */
 AP_Baro::AP_Baro()
 {
+    memset(sensors, 0, sizeof(sensors));
+    for(int i = 0 ; i < BARO_MAX_INSTANCES; i++) {
```

Comment:
please use uint8_t. We try not to use "int" as the size varies between platforms. I know in this case it is trivial, but its a good habit to get into.